unitTests/ui/Combobox/ComboboxValuesTest.ts

```diff
@@ -198,6 +229,16 @@ export function ComboboxValuesTest() {
         expect(isChildrenActiveAtIndex(0)).toBe(true);
       });
 
+      it(`when the last is active
+      should focus the first value`, () => {
+        comboboxValues.moveActiveValueDown();
+        comboboxValues.moveActiveValueDown();
+        comboboxValues.moveActiveValueDown();
+
+        expect(countChildrenFocusedAtIndex(0)).toEqual(2);
```

Comment:
Hmm, why is the number of children focused at index 0 equal to two? What are the two children in this case?

Reply:
I badly named this function. This counts how many times the child at a given index was focused. It's equal to two because the first child was focused twice. I'll rename it.